float64 latitude    # Latitude (degrees)
float64 longitude   # Longitude (degrees)
float64 height      # Height above WGS84 ellipsoid (meters)
float64 h_accuracy  # Horizontal position estimated standard deviation (meters)
float64 v_accuracy  # Vertical position estimated standard deviation (meters)
int8 n_satellites   # Number of satellites used in solution